### ISSUE TITLE
How can I get an annotated constructor?

### ISSUE THREAD

**Issue by tomas-langer** [classgraph/classgraph#447]
I am trying to get an annotated constructor. I use 
```java
ScanResult scan = new ClassGraph()
    .overrideClassLoaders(classLoader)
    .enableAllInfo()
    .scan();

ClassInfoList classesWithMethodAnnotation = scan.getClassesWithMethodAnnotation(reflectedClassName);

classesWithMethodAnnotation.forEach(it -> {
            it.getMethodAndConstructorInfo().forEach(method -> {
                if (method.hasAnnotation(reflectedClassName)) {
                    if (method.isConstructor()) {
                       // how do I get the Constructor instance?
                        context.register(it.loadClass()).add(method.loadClassAndGetMethod());
                    } else {
                        context.register(it.loadClass()).add(method.loadClassAndGetMethod());
                    }
                }
            });
        });
```

Unfortunatelly the `method.loadClassAndGetMethod()` returns `Method` and there is no `method.loadClassAndGetConstructor` to get an instance of `Constructor`

Is the only way to find it on the class myself through reflection?

The code above throws a `java.lang.IllegalArgumentException: No such method: ....MyClass.<init>`
as that is not a method, but a constructor...

Thanks!

**Comment by lukehutch** [comment]
Thanks for the report! This was an oversight. Can you please test the latest version in git?

**Comment by tomas-langer** [comment]
Tested and works as expected. Looking forward the next release!

**Comment by lukehutch** [comment]
Thanks for testing and reporting back. Released in 4.8.87.
